tagstudio/src/qt/helpers/file_deleter.py

```diff
@@ -0,0 +1,41 @@
+import logging
+import pathlib
+import subprocess
+import sys
+import traceback
+from pathlib import Path
+from typing import Callable
+
+ERROR = f"[ERROR]"
+WARNING = f"[WARNING]"
+INFO = f"[INFO]"
+
+logging.basicConfig(format="%(message)s", level=logging.INFO)
+
+
+def delete_file(path: str | Path, callback: Callable):
+    _path = str(path)
+    _file = Path(_path)
+    logging.info(f"Deleting file: {_path}")
+    if not _file.exists():
+        logging.error(f"File not found: {_path}")
+        return
+    try:
+        _file.unlink()
+        callback()
+    except:
+        traceback.print_exc()
+
+
+class FileDeleterHelper:
+    def __init__(self, filepath: str | Path):
+        self.filepath = str(filepath)
+
+    def set_filepath(self, filepath: str | Path):
+        self.filepath = str(filepath)
```

Comment:
no reason to convert these to `str` when it's later passed to `delete_file()` which expects `Path`